install/installer/traf_cloudera_mods98

```diff
@@ -158,6 +158,21 @@ if [ $? != 0 ]; then
 fi
 sudo su $HDFS_USER --command "$HADOOP_BIN_PATH/hadoop fs -chown -R $TRAF_USER:trafodion /lobs"
 
+
+#Create Backup directory 
+sudo su $HDFS_USER --command "$HADOOP_BIN_PATH/hadoop fs -mkdir /trafodion_backups" 2> $HOME/traf_temp_output
+if [ $? != 0 ]; then
+   # ok if directory already exists
+   dir_exists=$(grep "File exists" $HOME/traf_temp_output | wc -l)
```

Comment:
I think this code could be simplified by using -p option to the -mkdir command. Then it should not give error for directory exists.